model_cards/aliosm/ai-soco-c++-roberta-tiny/README.md

```diff
@@ -0,0 +1,55 @@
+---
+language: "c++"
+tags:
+- exbert
```

Comment:
Just a note that exbert support is not automatic, ie. cannot be added on the fly

We're thinking about enabling this in the future

cc @Narsil